src/ResourceManagement/BatchAI/BatchAIClusterImpl.cs

```diff
@@ -0,0 +1,360 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Linq;
```

Comment:
Please fix the document formatting. Ctr+A then Ctr+K,D.

Reply:
Fixed, and for other classes too.